purchases/src/main/kotlin/com/revenuecat/purchases/coroutinesExtensions.kt

```diff
@@ -10,13 +11,20 @@ import kotlin.coroutines.suspendCoroutine
  *
  * @warning This function is marked as [ExperimentalPreviewRevenueCatPurchasesAPI] and may change in the future.
  *
+ * @param fetchPolicy Specifies cache behavior for customer info retrieval (optional).
+ * Defaults to [CacheFetchPolicy.default]: [CACHED_OR_FETCHED].
+ *
  * @throws [PurchasesException] with a [PurchasesError] if there's an error retrieving the customer info.
  * @return The [CustomerInfo] or a [PurchasesException] with the [PurchasesError]
  */
+@JvmOverloads
 @ExperimentalPreviewRevenueCatPurchasesAPI
-suspend fun Purchases.awaitCustomerInfo(): CustomerInfo {
+suspend fun Purchases.awaitCustomerInfo(
+    fetchPolicy: CacheFetchPolicy = CacheFetchPolicy.default(),
```

Comment:
Could you add the usage of this in the `api-tests` module? It's been helpful to avoid regressions in the public API.